cinema/views.py

```diff
@@ -34,6 +39,28 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        if self.action == ("list", "retrieve"):
```

Comment:
The condition `self.action == ("list", "retrieve")` is incorrect. It should be `self.action in ("list", "retrieve")` to check if the action is either 'list' or 'retrieve'.